doc/theory/glossary.tex

```diff
@@ -396,54 +387,66 @@
 	
 	The name is inspired from the case when the persistence diagram comes from persistent homology.
 	
-	\subsection*{Distances, inner products and kernels} \label{metric_inner_product_and_kernel}
+	\subsection*{Metric space} \label{metric space}	
```

Comment:
As above, would you mind `metric_space` ?

Reply:
I will change it. Sorry for the slip.